acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/SwaggerUIAcceptanceTest.java

```diff
@@ -78,7 +78,7 @@ void web3signerYamlEndPointRespondsWith200() {
     given()
         .baseUri(signer.getUrl())
         .when()
-        .get(SWAGGER_UI_ENDPOINT + "/web3signer.yaml")
+        .get(SWAGGER_UI_ENDPOINT + "/eth2/web3signer.yaml")
```

Comment:
this is the failing test. Can't see exactly what's wrong with this though, at first glance seems like it should work 🤔

Reply:
Its Vertx 3.x handling of trailing / in route paths. I've added quick fix and a note in Runner class.

Reply:
ah I see, nice!